paper-ieee-trans.tex

```diff
@@ -63,61 +63,63 @@
 \section{Introduction}
 
 \IEEEPARstart{D}uring the last 15 years the provision of an European e-Infrastructure,
-which supports an unified access to large-scale computing and intense data analysis
+supporting an unified access to large-scale computing and intense data analysis
 applications, has been driven by the needs of scientific communities as part of their
-pan-European research collaborations. One of the well-known examples is the Worldwide 
-Large Hadron Collider (LHC) Computing Grid infrastructure, powered 
-by the particle physics scientific community, built up to satisfy the computing needs
-of processing, storing and analysing the huge quantity of data coming from the 
+pan-European research collaborations. One such example is the Worldwide 
+Large Hadron Collider (LHC) Computing Grid infrastructure, motivated 
+by the particle physics scientific community, build to satisfy the computing needs
```

Comment:
this one should be past right? --> built

Reply:
I think so